index.html

```diff
@@ -0,0 +1,250 @@
+        }
+
+        .main-content h1 {
+            font-family: Montserrat;
+            font-size: 58px;
+            font-style: normal;
+            font-weight: 800;
+            line-height: 80px;
+            letter-spacing: 0.2px;
+            max-width: 570px;
+
+        }
+
+        .main-content h4 {
+            font-family: Montserrat;
+            font-size: 20px;
+            font-style: normal;
+            font-weight: 500;
+            line-height: 30px;
+            letter-spacing: 0.2px;
+            max-width: 570px;
+
+        }
+
+        .btn {
+            display: flex;
+            margin-left: 30px;
+        }
+
+        .header_search {
+
+            background: none;
+            border: none;
+            width: 40px;
+
+        }
+
+        .header_cart {
+
+            background: none;
+            border: none;
+            width: 40px;
+        }
+
+        .btn-info {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            gap: 10px;
+        }
+
+        .btn-1,
+        .btn-2 {
+            padding: 10px 36px;
+            border-radius: 37px;
+            font-family: Montserrat;
+            font-size: 14px;
+            font-style: normal;
+            font-weight: 700;
+            line-height: 28px;
+            letter-spacing: 0.2px;
+        }
+
+        .btn-1 {
+
+
+            background: #FFA62B;
+
+        }
+
+        .btn-2 {
+
+            border: 1px solid #fff;
+        }
+
+        @media screen (max-width: 320px) {
+            .header_block {
+                align-items: center;
+                flex-direction: column;
+            }
+
+            .nav {
+                justify-content: center;
+                align-items: center;
+                flex-direction: column;
+            }
+        }
+     
+    </style>
+</head>
+
+<body>
+    <header class="header">
+        <div class="nav_menu">
+            <div class="container">
+                <div class="header_block">
+
+                    <a href="" class="header_logo">KMarketer</a>
+                    <div class="nav">
+                        <nav>
+                            <ul>
+                                <li><a href="">Home</a></li>
+                                <li><a href="">Product</a></li>
+                                <li><a href="">Pricing</a></li>
+                                <li><a href="">Contact</a></li>
+                            </ul>
+                        </nav>
+                        <div class="btn">
+                            <button type="submit" class="header_search"><img src="img/icn search .icn-xs.svg"></button>
+                            <button class="header_cart"><img src="img/icn shopping-cart .icn-xs.svg"></button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="top_banner">
+            <div class="container">
```

Comment:
You don't need to set the container twice. You could wrap all the content once.
Ex: 
`<section class="top-banner">`
  `<div class="container">`
    `<header></header>`
    `<div class="main-content">`
  `</div>`
`</section>`

This would be a more optimal layout